### PR TITLE
Upgrade stackblitz demos to use Angular 9

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

File: src/app/public/modules/code-examples/code-examples-editor.service.ts [hunk 1]
```diff
@@ -66,11 +65,11 @@ export class SkyDocsCodeExamplesEditorService {
       '@skyux/popovers': skyuxVersion,
       '@skyux/router': skyuxVersion,
       '@skyux/theme': skyuxVersion,
-      'core-js': '^2.6.5',
+      'core-js': '2',
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/code-examples/code-examples-editor.service.ts:68]: ☝️  https://github.com/stackblitz/core/issues/930

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-docs-angular-9-upgrade](https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-docs-angular-9-upgrade)

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-docs-angular-9-upgrade](https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-docs-angular-9-upgrade)

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-docs-angular-9-upgrade](https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-docs-angular-9-upgrade)

**Pull request merged**